docker-compose.yaml

```diff
@@ -21,6 +21,8 @@ services:
       - POSTGRES_DB=db-${DATABASE_DB}
     volumes:
       - db:/var/lib/postgresql/data
+    ports:
```

Comment:
I don't think this should be here?

Reply:
fair. i'm not quite sure what the right thing is here. there are two things i want:
1. as someone developing locally, i want to be able to run the scheduler or server using gradle (intellij so i can get a debugger). in order for that to work they need to be able to connect to the database by localhost:<some port>.
2. i want to from a terminal to be able to use psql to interact with a docker container. 

for both of these, the only way i know how to do this is to expose it on a port that is not the same port that postgres already uses on my local machine.

based on that would:
1. you be comfortable keeping this but i add a comment explaining why it's there. (potentially we should move it to a dev compose configuration)
2. have an idea on another way to achieve these goals?

Reply:
My concern was partially due to the fact that it's an unrelated change to the overall PR, and it could extremely rarely be a breaking change for a user upgrading who already has something running on that port. 

I like the dev configuration idea because it sounds like it solves your problem without any downsides.

Reply:
agreed.